frontend/src/pages/talks/TalkSingleEntry.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { Box, Paper, Typography } from '@mui/material';
+import { TalkEntry } from 'src/services/mocks';
+import { extractVideoId } from 'src/utils/video';
+import { useTranslation } from 'react-i18next';
+
+const TalkSingleEntry = ({ talk }: { talk: TalkEntry; compact?: boolean }) => {
+  const { i18n } = useTranslation();
+
+  const abstractParagraphs = talk.abstract.split('\n');
+
+  function isPast(talk: TalkEntry) {
+    return new Date(talk.date) < new Date();
+  }
+
+  const title = (
+    <Box display="flex" gap={2}>
+      <Typography variant="h4">{talk.title}</Typography>
+    </Box>
+  );
+
+  const formatDate = (dateString: string) => {
+    if (i18n.language == 'fr') {
+      const date = new Date(dateString);
+
+      const day = date.getDate();
+      const month = date.getMonth() + 1;
+      const year = date.getFullYear();
+
+      const formattedDate = `${day < 10 ? '0' + day : day}/${
+        month < 10 ? '0' + month : month
+      }/${year}`;
+
+      return formattedDate;
+    }
+    return dateString;
+  };
+
+  return (
+    <>
+      <a
+        className="no-decoration"
+        href={isPast(talk) ? talk.youtube_link : talk.invitation_link}
```

Comment:
I see a possible error here. It's possible that the talk is past without a YouTube link if no one took the time to upload the video.

A more robust way could be:
- if the talk is in the future AND there is an invitation link, we display the invitation link
- if the talk is in the past AND there is a YouTube link, we display the YouTube link
- else, no link

What do you think @bbosson ?